provision.yml

```diff
@@ -67,11 +66,12 @@
 
 - hosts: proxy
   roles:
-    - { role: satosa,      tags: ['proxy','satosa'    ] }
-    - { role: pyff,        tags: ['proxy','pyff'      ] }
-    - { role: cmservice,   tags: ['proxy','cmservice' ] }
-    - { role: mysql,       tags: ['proxy','mysql'     ] }
-    - { role: proxy_mysql, tags: ['proxy','mysql'     ] }
+    - { role: syslog-client, tags: ['proxy','syslog'    ] }
```

Comment:
Please move the syslog client to the common target, so that we get all syslog info from the hosts.